editor.planx.uk/src/@planx/components/PlanningConstraints/utils.test.ts

```diff
@@ -0,0 +1,145 @@
+import { defaultIntersectingConstraints, defaultNots, mockConstraints } from "./mocks/overrides/constraints";
+import { DEFAULT_FN } from "./model";
+import { handleOverrides } from "./utils";
+
+describe("handleOverrides basic", () => {
+  test("when no constraints are reported as inaccurate, it returns the original intersections and nots", () => {
+    const overrides = undefined;
+    const result = handleOverrides(DEFAULT_FN, mockConstraints, overrides, defaultIntersectingConstraints, defaultNots);
+
+    expect(result).toEqual({
+      intersectingConstraints: defaultIntersectingConstraints,
+      nots: defaultNots,
+    });
+  });
+
+  test.skip("when a constraint was reported as inaccurate, but is not part of the latest GIS API response, it's skipped", () => {
```

Comment:
This is a super edge casey thing that I _suspect_ can technically happen (because the component fetches fresh data on each load, but we also keep your previous overrides when coming back), but should be very unlikely. 

Mocked this test to not forget about it, but the `handleOverrides` function isn't handling this yet